recipes/perl-string-random/0.30/meta.yaml

```diff
@@ -1,15 +1,15 @@
 {% set name = "perl-string-random" %}
 {% set version = "0.30" %}
-{% set sha256 = "81139b9b96d50e0e409b6de10f4ffa816c84460d9c8a158902548dc2a4bfad1b" %}
+{% set hash = "81139b9b96d50e0e409b6de10f4ffa816c84460d9c8a158902548dc2a4bfad1b" %}
```

Comment:
can we leave sah256?

Reply:
it's done

Reply:
Thanks for merge but built failed .how can I retry ?